actions/05nn.json

```diff
@@ -0,0 +1,30 @@
+  
+{
+    "date": "2020/03/17",
+    "sources": [
+        "https://medium.com/@amazonemployeesclimatejustice/coronavirus-amazon-log-cfdc9a6f4437",
+        "https://medium.com/@amazoniansunitednyc/we-amazon-workers-demand-coronavirus-protections-16f28ad8b15f"
+    ],
+    "actions": [
+        "open_letter"
+    ],
+    "struggles": [
+        "working_conditions"
+    ],
+    "employment_types": [
+        "blue_collar_workers",
+        "white_collar_workers"
+    ],
+    "description": "Over 5000 Amazon employees signed an open letter to Jeff Bezos calling for improved working conditions during the coronavirus pandemic, including paid sick leave regardless of diagnosis, child care, and 1.5x hazard pay. Since then, on March 22nd, the letter has been updated to include a call for transparency on any diagnosed covid-19 cases at the facilities, a halt on the shipping of non-essential items, and medical coverage to part-time employees. On March 27th, Amazon Employees for Climate Justice emailed its constituents (mostly corporate employees) to encourage them to sign the open letter, which purportedly lead to an additional 580 signatures.",
+    "locations": [
+        "usa"
+    ],
+    "companies": [
+        "amazon"
+    ],
+    "workers": 5210,
+    "tags": [
+        "coronavirus"
```

Comment:
should add coworker solidarity tag too i believe (amazon employees for climate justice is mainly white-collar/in-house)